.changeset/soft-horses-ring.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/admin-theme": major
+---
+
+Remove `secondary` color styling from `Checkbox` and `Radio`
```

Comment:
Technically the MUI styling still exists for this variant, it's just our custom styling that is being removed

```suggestion
Remove custom `secondary` color styling from `Checkbox` and `Radio`
```